### PR TITLE
Add new ssh2_disconnect stub

### PR DESCRIPTION
**Pull request by CyberLine**

### CODE REVIEW

File: ssh2/ssh2.php [hunk 1]
```diff
@@ -177,6 +177,18 @@
...
 
+/**
+ * (PECL ssh2 &gt;= 1.0)<br/>
+ * Close a connection to a remote SSH server
+ * @link http://php.net/manual/en/function.ssh2-disconnect.php
+ * @param resource $session <p>
+ * An SSH connection link identifier, obtained from a call to
+ * ssh2_connect.
+ * </p>
+ * return bool
```
Comment (b1rdex) [inline ssh2/ssh2.php]: `@return bool`

File: ssh2/ssh2.php [hunk 2]
```diff
@@ -177,6 +177,18 @@
  */
 function ssh2_connect ($host, $port = null, array $methods = null , array $callbacks = null ) {}
 
+/**
+ * (PECL ssh2 &gt;= 1.0)<br/>
+ * Close a connection to a remote SSH server
+ * @link http://php.net/manual/en/function.ssh2-disconnect.php
```
Comment (b1rdex) [inline ssh2/ssh2.php]: Please use https link

File: ssh2/ssh2.php (hunk 1 above)
Comment (CyberLine) [reply ssh2/ssh2.php]: mh, the PR is missing the latest commit ...

Comment (b1rdex) [reply ssh2/ssh2.php]: That's strange. Commit view gives 404 too: https://github.com/JetBrains/phpstorm-stubs/commit/00b3da8f2f2090a5edffc4f89d5a4ee79b6c5a21

Comment (b1rdex) [reply ssh2/ssh2.php]: Link opens now, no 404 anymore. But commit is still not present in PR. Could you please try to rebase/force push branch or recreate PR?

**Pull request closed without merging**